### PR TITLE
Questlines

### PR DESCRIPTION
**Pull request by Aegyo**
I've tested this a little and it seems to work, but I wouldn't be surprised if I've missed something and it breaks spectacularly.

The tutorial quests might need changing a little, particularly when you have a quest to go kill stuff on route 2, but you can't access it yet.

### CODE REVIEW

**Comment by dennism1997** [comment]
Looking good! I think there should also be a quest that tells you to defeat 10 pokemon on Route 1

File: src/components/questDisplay.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <div class="page-item" style="background-color: white; min-height: 100px">
     <br>
     <button style="vertical-align: middle;" class='btn btn-primary'
-            data-bind='visible: !player.currentQuest(),
+            data-bind='visible: (player.tutorialProgress() == 6) && !player.currentQuest(),
```
Comment (dennism1997) [inline src/components/questDisplay.html]: Is 6 the final tutorial quest? A constant is better then, or maybe a function that counts them

**Comment by Ishadijcks** [comment]
Small bug where the progressbar of the tutorial shows 0/1 instead of the real progress.
If you can fix this and the magic number I aprove, it looks good :)

**Comment by Ishadijcks** [comment]
Great

**Pull request merged**